resources.go

```diff
@@ -112,7 +113,14 @@ func Provider() tfbridge.ProviderInfo {
 					},
 				},
 			},
-			"digitalocean_domain": {Tok: digitalOceanResource(digitalOceanMod, "Domain")},
+			"digitalocean_domain": {
+				Tok: digitalOceanResource(digitalOceanMod, "Domain"),
+				Fields: map[string]*tfbridge.SchemaInfo{
+					"name": tfbridge.AutoNameTransform("name", 63, func(name string) string {
+						return strings.ToLower(name)
+					}),
```

Comment:
This doesn’t look like it accomplishes what the description says. 

Are you trying to make `name` required?

Reply:
ok, I got it :)

I just managed to get the following into play

```
+        if name is None:
+            raise TypeError("Missing required property 'name'")
         __props__['name'] = name
```

This is exactly the behaviour I wanted